light-curve/light_curve/light_curve_py/features/rainbow/_base.py

```diff
@@ -269,18 +271,39 @@ def _eval(self, *, t, m, sigma, band):
             y=m,
             yerror=sigma,
         )
-        minuit = self.Minuit(least_squares, **initial_guesses)
-        minuit.migrad()
+        self.minuit = self.Minuit(least_squares, **initial_guesses)
+        # TODO: expose these parameters through function arguments
+        self.minuit.print_level = 1
+        self.minuit.strategy = 2
+        self.minuit.migrad(ncall=10000, iterate=10)
 
-        reduced_chi2 = minuit.fval / (len(t) - self.size)
-        params = np.array(minuit.values)
+        self.valid = self.minuit.valid
+        reduced_chi2 = self.minuit.fval / (len(t) - self.size)
+        params = np.array(self.minuit.values)
+        self.errors = np.array(self.minuit.errors)
 
         self._unscale_parameters(params, t_scaler, m_scaler)
         if self.with_baseline:
             self._unscale_baseline_parameters(params, m_scaler)
 
+        # Unscale errors
+        # FIXME: is there any better but generic way to unscale all relevant errors without shifting?..
```

Comment:
`Scaler` has `undo_scale` which doesn't shift
https://github.com/light-curve/light-curve-python/blob/bc1039db2569919d6ff2751782faf6f129cf12c9/light-curve/light_curve/light_curve_py/features/rainbow/_scaler.py#L47-L48

Reply:
Yes, but there is no way to use it without knowing the underlying data - `_unscale_parameters` method is defined in subclasses, and there is no way to know which parameter was transformed using which scaler from the base class apart from using it. So there is no clean way to apply just `undo_scale` method if the subclass uses `undo_shift_scale` (or even `undo_shift_scale_band`). Well, of course apart from defining new virtual method like `_unscale_parameters_errors` for the subclasses to implement - but that's probably too much?..

Reply:
Ah, I see now what did you mean, it is about parameter errors, not about flux errors. Sorry for the confusion